# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296103880 28800
#      Wed Jan 26 20:51:20 2011 -0800
# Node ID 2090995ca588feaf1f63697c791094d369a97eaf
# Parent  db1f49eaba6b4fb787c5aa0df902c4efda706cba
Correct en-dash,em-dash instances in docstrings.
Add a few more operators to operator index.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,16 +1,23 @@
 2011-01-26  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/mk_doc_cache.m, interpreter/mkoctfile.1,
 	interpreter/munge-texi.cc, interpreter/doccheck/mk_undocumented_list:
 	Strip trailing whitespace.
 
 2011-01-26  Rik  <octave@nomad.inbox5.com>
 
+	* interpreter/basics.txi: Use @code to eliminate extra single quotes
+	in index.
+	* interpreter/container.txi: Add @opindex citations for '.', '{' and
+	'}'.
+
+2011-01-26  Rik  <octave@nomad.inbox5.com>
+
 	* interpreter/octave-bug.1: Delete obsolete man page
 	* interpreter/Makefile.am: Stop distributing octave-bug.1 man page.
 
 2011-01-26  Olaf Till <olaf.till@uni-jena.de>
 
 	* interpreter/package.txi: Document variables MKOCTFILE,
 	OCTAVE_CONFIG, and OCTAVE exported by pkg for configure and make.
 
diff --git a/doc/interpreter/basics.txi b/doc/interpreter/basics.txi
--- a/doc/interpreter/basics.txi
+++ b/doc/interpreter/basics.txi
@@ -669,17 +669,17 @@ is saved.
 @DOCSTRING(history_size)
 
 @DOCSTRING(history_timestamp_format_string)
 
 @DOCSTRING(EDITOR)
 
 @node Customizing readline
 @subsection Customizing @code{readline}
-@cindex @file{~/.inputrc}
+@cindex @code{~/.inputrc}
 @cindex customizing @code{readline}
 @cindex @code{readline} customization
 
 Octave uses the GNU Readline library for command-line editing and
 history features.  Readline is very flexible and can be modified through
 a configuration file of commands (See the GNU Readline library for the
 exact command syntax).  The default configuration file is normally
 @file{~/.inputrc}.
diff --git a/doc/interpreter/container.txi b/doc/interpreter/container.txi
--- a/doc/interpreter/container.txi
+++ b/doc/interpreter/container.txi
@@ -61,19 +61,22 @@ expressions
 @example
 @group
 x.a = 1;
 x.b = [1, 2; 3, 4];
 x.c = "string";
 @end group
 @end example
 
+@opindex .
 @noindent
-create a structure with three elements.  To print the value of the
-structure, you can type its name, just as for any other variable:
+create a structure with three elements.  The @samp{.} character separates
+the structure name from the field name and indicates to Octave that this
+variable is a structure.  To print the value of the
+structure you can type its name, just as for any other variable:
 
 @example
 @group
 x
      @result{} x =
         @{
           a = 1
           b =
@@ -489,17 +492,18 @@ and @samp{@}} as allocation and indexing
 * Creating Cell Arrays::
 * Indexing Cell Arrays::
 * Cell Arrays of Strings::
 * Processing Data in Cell Arrays::
 @end menu
 
 @node Basic Usage of Cell Arrays
 @subsection Basic Usage of Cell Arrays
-
+@opindex @{
+@opindex @}
 As an example, the following code creates a cell array containing a
 string and a 2-by-2 random matrix
 
 @example
 c = @{"a string", rand(2, 2)@};
 @end example
 
 @noindent
diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,16 @@
+2011-01-26  Rik  <octave@nomad.inbox5.com>
+
+	* scripts/general/interp1.m, scripts/general/interp2.m,
+	scripts/general/interp3.m, scripts/general/interpn.m: Use em-dash
+	rather than en-dash.
+	* scripts/miscellaneous/mkoctfile.m: Use @code to prevent option
+	turning to en-dash.
+
 2011-01-26  John W. Eaton  <jwe@octave.org>
 
 	* mkdoc: Untabify.
 
 	* mkdoc, gethelp.cc: Strip trailing whitespace.
 
 2011-01-26  John W. Eaton  <jwe@octave.org>
 
diff --git a/scripts/general/interp1.m b/scripts/general/interp1.m
--- a/scripts/general/interp1.m
+++ b/scripts/general/interp1.m
@@ -41,17 +41,17 @@
 ##
 ## @item 'pchip'
 ## Piecewise cubic Hermite interpolating polynomial
 ##
 ## @item 'cubic'
 ## Cubic interpolation from four nearest neighbors
 ##
 ## @item 'spline'
-## Cubic spline interpolation--smooth first and second derivatives
+## Cubic spline interpolation---smooth first and second derivatives
 ## throughout the curve
 ## @end table
 ##
 ## Appending '*' to the start of the above method forces @code{interp1}
 ## to assume that @var{x} is uniformly spaced, and only @code{@var{x}
 ## (1)} and @code{@var{x} (2)} are referenced.  This is usually faster,
 ## and is never slower.  The default method is 'linear'.
 ##
diff --git a/scripts/general/interp2.m b/scripts/general/interp2.m
--- a/scripts/general/interp2.m
+++ b/scripts/general/interp2.m
@@ -60,17 +60,17 @@
 ##
 ## @item 'pchip'
 ## Piecewise cubic Hermite interpolating polynomial.
 ##
 ## @item 'cubic'
 ## Cubic interpolation from four nearest neighbors.
 ##
 ## @item 'spline'
-## Cubic spline interpolation--smooth first and second derivatives
+## Cubic spline interpolation---smooth first and second derivatives
 ## throughout the curve.
 ## @end table
 ##
 ## If a scalar value @var{extrapval} is defined as the final value, then
 ## values outside the mesh as set to this value.  Note that in this case
 ## @var{method} must be defined as well.  If @var{extrapval} is not
 ## defined then NA is assumed.
 ##
diff --git a/scripts/general/interp3.m b/scripts/general/interp3.m
--- a/scripts/general/interp3.m
+++ b/scripts/general/interp3.m
@@ -47,17 +47,17 @@
 ##
 ## @item 'linear'
 ## Linear interpolation from nearest neighbors.
 ##
 ## @item 'cubic'
 ## Cubic interpolation from four nearest neighbors (not implemented yet).
 ##
 ## @item 'spline'
-## Cubic spline interpolation--smooth first and second derivatives
+## Cubic spline interpolation---smooth first and second derivatives
 ## throughout the curve.
 ## @end table
 ##
 ## The default method is 'linear'.
 ##
 ## If @var{extrap} is the string 'extrap', then extrapolate values beyond
 ## the endpoints.  If @var{extrap} is a number, replace values beyond the
 ## endpoints with that number.  If @var{extrap} is missing, assume NA.
diff --git a/scripts/general/interpn.m b/scripts/general/interpn.m
--- a/scripts/general/interpn.m
+++ b/scripts/general/interpn.m
@@ -47,17 +47,17 @@
 ##
 ## @item 'linear'
 ## Linear interpolation from nearest neighbors.
 ##
 ## @item 'cubic'
 ## Cubic interpolation from four nearest neighbors (not implemented yet).
 ##
 ## @item 'spline'
-## Cubic spline interpolation--smooth first and second derivatives
+## Cubic spline interpolation---smooth first and second derivatives
 ## throughout the curve.
 ## @end table
 ##
 ## The default method is 'linear'.
 ##
 ## If @var{extrapval} is the scalar value, use it to replace the values
 ## beyond the endpoints with that number.  If @var{extrapval} is missing,
 ## assume NA.
diff --git a/scripts/miscellaneous/mkoctfile.m b/scripts/miscellaneous/mkoctfile.m
--- a/scripts/miscellaneous/mkoctfile.m
+++ b/scripts/miscellaneous/mkoctfile.m
@@ -62,17 +62,17 @@
 ## Compile but do not link.
 ##
 ## @item -g
 ## Enable debugging options for compilers.
 ##
 ## @item -o FILE
 ## @itemx --output FILE
 ## Output file name.  Default extension is .oct
-## (or .mex if --mex is specified) unless linking
+## (or .mex if @samp{--mex} is specified) unless linking
 ## a stand-alone executable.
 ##
 ## @item -p VAR
 ## @itemx --print VAR
 ## Print the configuration variable VAR@.  Recognized variables are:
 ##
 ## @example
 ##    ALL_CFLAGS                FFTW_LIBS
